### PR TITLE
remove method 'setEP'

### PR DESCRIPTION
**Pull request by Lucy0**
I removed the method setEP

### CODE REVIEW

File: ChessBoard.py [hunk 1]
```diff
@@ -176,8 +176,8 @@ def updateKingLocations(self):
                 if self._board[y][x] == "kkk":
                     self._black_king_location = (x, y)
 
-    def setEP(self, epPos):
```
Comment (nsomar) [inline ChessBoard.py:179]: Why did you remove this method? please explain the rationale behind it.

Comment (Lucy0) [inline ChessBoard.py:179]: Remove it for assignment question 8
